tests/plugins/test_default_bucket.py

```diff
@@ -338,6 +338,20 @@ def test_events_sent_on_bucket_and_collection_creation(self):
         assert _events[2].payload['uri'] == records_uri.replace('default',
                                                                 bucket_id)
 
+    def test_second_call_on_default_bucket_doesnt_send_create_events(self):
+        bucket_url = '/buckets/default'
+        self.app.get(bucket_url, headers=self.headers)
+        assert len(_events) == 1
+        self.app.get(bucket_url, headers=self.headers)
+        assert len(_events) == 1
+
+    def test_second_call_on_default_bucket_collection_doesnt_send_create_events(self):
+        collection_url = '/buckets/default/collections/articles'
+        self.app.get(collection_url, headers=self.headers)
+        assert len(_events) == 2
+        self.app.get(collection_url, headers=self.headers)
+        assert len(_events) == 2
```

Comment:
They're the same create events from `test_an_event_is_sent_on_implicit_bucket_creation` and `test_an_event_is_sent_on_implicit_collection_creation`. Since they're already there, I didn't think it was worth rehashing them. But a comment is a good idea.